api/v1alpha1/release_webhook_test.go

```diff
@@ -75,4 +76,14 @@ var _ = Describe("Release validation webhook", func() {
 			Expect(k8sClient.Update(ctx, release)).ShouldNot(HaveOccurred())
 		})
 	})
+
+	Describe("ValidateDelete() unit test.", func() {
+		It("Release.ValidateDelete() should return nil", func() {
```

Comment:
Sorry @parasense I noticed now my comments on these last files from some reason did not went through.  Same thing about the `It` description starting on lowercase.